src/blockchain/ain/jellyfish/services/jellyfish.service.ts

```diff
@@ -0,0 +1,94 @@
+import { MainNet, Network, TestNet } from '@defichain/jellyfish-network';
+import { CTransactionSegWit, Vout, toOPCodes } from '@defichain/jellyfish-transaction';
+import { Prevout } from '@defichain/jellyfish-transaction-builder';
+import { JellyfishWallet, WalletAccount, WalletHdNode } from '@defichain/jellyfish-wallet';
+import { Bip32Options, MnemonicHdNodeProvider } from '@defichain/jellyfish-wallet-mnemonic';
+import { WhaleWalletAccount, WhaleWalletAccountProvider } from '@defichain/whale-api-wallet';
+import { Injectable } from '@nestjs/common';
+import { SchedulerRegistry } from '@nestjs/schedule';
+import BigNumber from 'bignumber.js';
+import { Config } from 'src/config/config';
+import { QueueHandler } from 'src/shared/queue-handler';
+import { RawTxDto } from '../dto/raw-tx.dto';
+import { RawTxUtil } from '../utils/raw-tx-util';
+import { UtxoProviderService } from './utxo-provider.service';
+import { SmartBuffer } from 'smart-buffer';
+import { RawTxMasternode } from '../utils/raw-tx-masternode';
+import { RawTxVault } from '../utils/raw-tx-vault';
+import { RawTxUtxo } from '../utils/raw-tx-utxo';
+import { RawTxAccount } from '../utils/raw-tx-account';
+import { RawTxPool } from '../utils/raw-tx-pool';
+
+@Injectable()
+export class JellyfishService {
```

Comment:
IMO the contents of this class are a bit mixed. I would prefer a dedicated RawTxService and move the rest out. Then it also get's clearer, what the `unlock` method is for. Maybe rename it to `unlockUtxos`?